CMIP6Plus/TermsOfUse/TermsOfUse6-5.md

```diff
@@ -23,16 +23,12 @@ The licensing agreements governing CMIP6Plus data depend on the model but genera
 
 ### General Warning:
 
-The CMIP6Plus archive contains the output of scientific simulations of the past and potential future that are subject to multiple sources of error, ranging from errors in data handling, to errors in the representation of the real world in either the model, or the experimental setup for which the model was used. Different parts of the CMIP6Plus archive may be subject to differing levels of such errors, and users should be alert to these issues, and their potential consequences (and to the limitations of liability expressed in the data license).
+The CMIP6Plus archive contains the output of scientific simulations of the past and potential future that are subject to multiple sources of error, ranging from errors in data handling to errors in the representation of the real world in either the model or the experimental setup for which the model was used. Different parts of the CMIP6Plus archive may be subject to differing levels of such errors, and users should be alert to these issues and their potential consequences (and to the limitations of liability expressed in the data licenses).
 
-###### Document version: 6.5 (20 November 2023)
+###### Document version: 6.5 (21 November 2023)
 
 [CC BY 4.0]: https://creativecommons.org/licenses/by/4.0/
-[CMIP6 Climate Model Data license page]: https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_source_id_licenses.html
-[CMIP6 GMD special issue]: http://www.geosci-model-dev.net/special_issue590.html
 [Data Citation Guidelines]: http://bit.ly/2gBCuqM
-[html rendering (model names)]: https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_source_id.html
-[CMIP6 source_id controlled vocabulary]: https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_source_id.json
-[html rendering (institution names)]: https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_institution_id.html
-[CMIP6 institution_id controlled vocabulary]: https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_institution_id.json
-[CMIP Publications webpage]: https://cmip-publications.llnl.gov/view/CMIP6/
+[here (model names)]: https://wcrp-cmip.github.io/CMIP6Plus_CVs/docs/CMIP6Plus_source_id.html
+[here (institution names)]: https://wcrp-cmip.github.io/CMIP6Plus_CVs/docs/CMIP6Plus_institution_id.html
```

Comment:
These links are 404.

Reply:
This is linked to work in progress as per https://github.com/WCRP-CMIP/CMIP6Plus_CVs/issues/5